Marlin/ultralcd_impl_HD44780.h

```diff
@@ -530,13 +530,15 @@ void lcd_print(char c) { charset_mapper(c); }
       safe_delay(2000);
     #endif
 
-    lcd_implementation_clear();
+    /*
```

Comment:
This is needed and can't be replaced with "nothing". If you do this you will not have special characters on the LCD anymore. Did you test that?